integ/components/deadline/deadline_01_repository/test/deadline_01_repository.test.ts

```diff
@@ -12,15 +12,15 @@ import awaitSsmCommand from '../../common/functions/awaitSsmCommand';
 const testingStackName = 'RFDKInteg-DL-TestingTier' + process.env.INTEG_STACK_TAG?.toString();
 const deadlineVersion = process.env.DEADLINE_VERSION?.toString();
 
-jest.setTimeout(10000);
+jest.setTimeout(60000);
```

Comment:
You made this comment in the PR itself:
> I did note that at least the WorkerFleet test can run up against jest's timeout limit and be in danger of failing. Since the tests make so many calls and stuff I decided to take care of it by adding one more update that ups the timeout limit on all of the jest tests to 60 seconds.

Maybe we could even add it as a comment here, so we don't remove/decrease this timeout in the future.

Reply:
That makes sense. The other option is to code the timeout into the `jest.config` file, then it wouldn't have to be set on a per-test basis but could still be overridden if necessary. I think I'll just do that.

Reply:
I can't find the place where we are setting this timeout now. Am I missing something?

Reply:
Accidentally didn't include it in the commit, which I will now correct. It's set in `jest.config.js`.